apps/controllo-ore-x/src/app/modules/auth/modules/project/modules/release/dialogs/release-dialog/release.dialog.html

```diff
@@ -0,0 +1,97 @@
+<lib-dialog-template
+  [cTitle]="title"
+  [isFooterEnabled]="true"
+  (closeDialog)="cancel()"
+  [isLoading]="isLoading"
+  [isError]="hasErrors"
+>
+  <div body>
+    <form class="-form p-0" [formGroup]="formHelper.form">
+      <div class="row mb-2 -form__fields">
+        <mat-form-field class="col" appearance="outline">
+          <mat-label>Versione</mat-label>
+          <input matInput placeholder="Versione" formControlName="version" />
+          <mat-error *ngFor="let error of [RT_FORM_ERRORS['REQUIRED']]">
+            <div
+              *ngIf="formHelper.form.controls['version'].hasError(error.name)"
+            >
+              {{ error.message }}
+            </div>
+          </mat-error>
+        </mat-form-field>
+
+        <mat-form-field class="col" appearance="outline">
+          <mat-label>Budget ore</mat-label>
+          <input
+            type="number"
+            matInput
+            placeholder="Budget ore"
+            formControlName="hoursBudget"
+            min="0"
+          />
+          <mat-error *ngFor="let error of [RT_FORM_ERRORS['REQUIRED']]">
+            <div
+              *ngIf="formHelper.form.controls['hoursBudget'].hasError(error.name)"
+            >
+              {{ error.message }}
+            </div>
+          </mat-error>
+        </mat-form-field>
+
+        <mat-form-field class="col" appearance="outline">
+          <mat-label>Budget ore fatturabili</mat-label>
+          <input
+            type="number"
+            matInput
+            placeholder="Budget ore fatturabili"
+            formControlName="billableHoursBudget"
+            min="0"
+          />
+          <mat-error *ngFor="let error of [RT_FORM_ERRORS['REQUIRED']]">
+            <div
+              *ngIf="formHelper.form.controls['billableHoursBudget'].hasError(error.name)"
```

Comment:
Make a function to unwrap the error name
```suggestion
              *ngIf="getFormControlError('billableHoursBudget')"
```